Python/Tests/Utilities.Python.Analysis/MockPythonInterpreter.cs

```diff
@@ -23,7 +23,7 @@
 using Microsoft.PythonTools.Interpreter;
 
 namespace TestUtilities.Python {
-    public class MockPythonInterpreter : IPythonInterpreter {
+    public class MockPythonInterpreter : IPythonInterpreter2 {
```

Comment:
I expected `AstPythonInterpreter` to implement this too, yes?

Reply:
Yes, I guess we are not calling it from outside just yet.